vmas/simulator/environment/environment.py

```diff
@@ -740,6 +740,51 @@ def render(
                 pos[Y] + cam_range[Y],
             )
 
+        # Include boundaries in the rendering
+        from vmas.simulator import rendering
+        from vmas.simulator.rendering import Line
+        from vmas.simulator.utils import Color
+
+        # Check if the world dimensions are defined
+        if self.world.x_semidim is not None and self.world.y_semidim is not None:
+            # Get the radius of the agents
+            radius = self.agents[0].shape.radius
+
+            # Get the origin coordinates and dimensions for rendering
+            origin_x, origin_y = (
+                self.scenario.render_origin[X],
+                self.scenario.render_origin[Y],
+            )
+            # Extend the dimensions to account for the agent's radius
+            x_semi, y_semi = (
+                self.world.x_semidim + radius,
+                self.world.y_semidim + radius,
+            )
+
+            # Define the corner points of the boundary
+            corners = [
+                (origin_x - x_semi, origin_y + y_semi),  # top_left
```

Comment:
Are you sure about this? the origin of the rendering is where the camera is, but the boundaries are always from the physical origin

Reply:
I've conducted some tests, and it turns out that the choice of 
```python
origin_x, origin_y = (0, 0)
```
would be the correct approach.

Reply:
meaning we can remove the origin from there